app/services/katello/pulp3/repository/apt.rb

```diff
@@ -47,9 +47,8 @@ def partial_repo_path
           "/pulp/deb/#{repo.relative_path}/".sub('//', '/')
         end
 
-        def copy_content_for_source
-          # TODO
-          fail NotImplementedError
+        def copy_content_for_source(source_repository, _options = {})
+          copy_units_by_href(source_repository.files.pluck(:pulp_id))
```

Comment:
shouldn't thinks be '.debs'

Reply:
Yep it should! Not sure why it looks like the debs got copied anyway...